pkg/model/history/resourcepath/apacheairflow.go

```diff
@@ -18,20 +18,20 @@ import (
 	"github.com/GoogleCloudPlatform/khi/pkg/model"
 )
 
-// composer#taskinstance#DAGID#RUNID#TASKID-MAPINDEX
-func ComposerTaskInstance(ti *model.AirflowTaskInstance) ResourcePath {
+// airflow#taskinstance#DAGID#RUNID#TASKID-MAPINDEX
+func AirflowTaskInstance(ti *model.AirflowTaskInstance) ResourcePath {
 	var detail = ti.TaskId()
 	if ti.MapIndex() != "-1" {
 		detail += "+" + ti.MapIndex()
 	}
-	return SubresourceLayerGeneralItem("Cloud Composer", "Task Instance", ti.DagId(), ti.RunId(), detail)
+	return SubresourceLayerGeneralItem("Apache Airflow", "TaskInstance", ti.DagId(), ti.RunId(), detail)
```

Comment:
If the resource is a subresource, you may need to consider adding `ParentRelationship` of it.
https://github.com/GoogleCloudPlatform/khi/blob/main/pkg/model/history/resourcepath/controlplane.go#L27